### PR TITLE
Add constants from <include/linux/sched.h>

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Needed for `sysinfo` as usual. :)

Constants come from https://github.com/torvalds/linux/blob/master/include/linux/sched.h.

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
I had to ignore the constants because they are not defined in `linux/sched.h`, no idea why. It's strange because it's the same file but only up to a certain point...

File: libc-test/build.rs [hunk 1]
```diff
@@ -4056,6 +4056,33 @@ fn test_linux(target: &str) {
                 true
             }
 
+            // FIXME: seems to not be available all the time (from <linux/include/sched.h>:
```
Comment (JohnTitor) [inline libc-test/build.rs:4059]:
```suggestion
            // FIXME: seems to not be available all the time (from <include/linux/sched.h>:
```

Seems swapped, but yeah, it's weird that 23.10 couldn't find them which has the v6.5 kernel header 🤔

**Review by JohnTitor — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
Thanks for the comment fix and the quick review!

**Pull request merged**